README.md

```diff
@@ -36,94 +35,28 @@ Here's a list of benefits you will get if you use this project:
   - For instance you can show: slowest jobs, most failing jobs, jobs requirying re-runs, repos consuming most CI
   - Some of the main tags by which you can slice the data: workflow file, repo, branch, commit, job_status, pull_request, run_attempt
 
-
 ## Do you want to try this?
 
-**NOTE**: If we invested a bit more on this project we would reduce the following steps to just install this Github App and add this configuration file somewhere to be determined.
-
 Steps to follow:
 
-- Fork the app
-- Create a Docker build
-- Deploy the image to your provider
-  - If you don't use GCP you may need to make some changes to the code
-- Create a Github App and install it in all your repos
-  - Instructions are provided below
-
-Please, give us some feedback in [this issue](https://github.com/getsentry/sentry-github-actions-app/issues/46).
-
-## Code
-
-Code explanation:
-
-- `github_sdk.py` has the generic logic to submit Github jobs as Sentry transactions. This module could be released separatedly.
-- `github_app.py` contains code to handle a Github App installation.
-- `web_app_handler.py`: Web app logic goes here.
-- `main.py` contains the code to respond to webhook events.
-
-## Set Up
-
-**NOTE**: Currently, this app is only used internally. See [milestone](https://github.com/getsentry/sentry-github-actions-app/milestone/1) for required work to support ingesting data from other orgs.
-
-These are the steps you will need to set up this backend and Github app for your organization.
-
-The steps of the next two sections are related. You will be generating keys and variables that are need to configure each component (the backend and the Github App), thus, you will need to go back and forth.
-
-### Sentry SaaS (or self-hosted Sentry)
+- Create a Sentry project to gather your Github CI data
+  - You don't need to select a platform in the wizard or any alerts
+  - Find the DSN key under the settings of the project you created
+- Create a public repo name `.sentry`
+- Create a file in that repo called `sentry_config.ini` with these contents (adjust your DSN value)
```

Comment:
These are the new steps to test out the GH App. We will need a link to the public version of the app.